Clean up dependencies for sequoia-openpgp:

backtrace: https://gitlab.com/sequoia-pgp/sequoia/-/merge_requests/881
base64: no clear reason for upgrading 0.12 to 0.13, and 0.13 implicates
        grep-printer, pem, plist, and ron in debian
lalrpop: upgrade to 0.19 goes through NEW b/c of new "lexer" feature,
         Neal Walfield said nothing after 0.17 was used
memsec: upgrade to 0.6 goes through NEW b/c of new "getrandom" feature

drop all crypto-crng and windows stuff

And we fake out the "vendored-nettle" thing, since on debian,
nettle is linked dynamically

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -32,10 +32,10 @@
 version = "1"
 
 [dependencies.backtrace]
-version = "0.3.46"
+version = "0.3.44"
 
 [dependencies.base64]
-version = "0.13"
+version = "0.12"
 
 [dependencies.buffered-reader]
 version = "0.20"
@@ -56,7 +56,7 @@
 version = "0.2"
 
 [dependencies.lalrpop-util]
-version = "0.19"
+version = "0.17"
 
 [dependencies.lazy_static]
 version = "1.3"
@@ -65,7 +65,7 @@
 version = "0.2"
 
 [dependencies.memsec]
-version = "0.6"
+version = "0.5.6"
 default-features = false
 
 [dependencies.nettle]
@@ -91,38 +91,16 @@
 [dev-dependencies.rpassword]
 version = "5.0"
 [build-dependencies.lalrpop]
-version = "0.19"
+version = "0.17"
 
 [features]
 compression = ["compression-deflate", "compression-bzip2"]
 compression-bzip2 = ["bzip2", "buffered-reader/compression-bzip2"]
 compression-deflate = ["flate2", "buffered-reader/compression-deflate"]
-crypto-cng = ["winapi", "win-crypto-ng", "ed25519-dalek", "num-bigint-dig"]
 crypto-nettle = ["nettle"]
 default = ["compression", "crypto-nettle"]
 vendored = ["vendored-nettle"]
-vendored-nettle = ["nettle/vendored"]
-[target."cfg(windows)".dependencies.ed25519-dalek]
-version = "1"
-features = ["rand", "u64_backend"]
-optional = true
-default-features = false
-
-[target."cfg(windows)".dependencies.num-bigint-dig]
-version = "0.6"
-optional = true
-default-features = false
-
-[target."cfg(windows)".dependencies.win-crypto-ng]
-version = "0.2"
-features = ["rand"]
-optional = true
-
-[target."cfg(windows)".dependencies.winapi]
-version = "0.3"
-features = ["bcrypt"]
-optional = true
-default-features = false
+vendored-nettle = []
 [badges.gitlab]
 repository = "sequoia-pgp/sequoia"
 
